static/partials/seats.html

```diff
@@ -17,11 +17,11 @@ <h1 ng-show="vm.overview">Miejsca uczestników</h1>
                                 <input ng-click="vm.take_or_relese_seat(raw, number, user)"
                                        type="checkbox"
                                        id="{{ raw }}{{ number }}"
-                                       ng-disabled="vm.user_seat.users && (vm.user_seat.row != raw || vm.user_seat.number != number) || user.empty_raw"
+                                       ng-disabled="vm.user_seat.users && (vm.user_seat.row != raw || vm.user_seat.number != number) || user.empty_row"
```

Comment:
i think this needs some adjusments in BE ?

Reply:
I think this is done in BE? (in the same commit, btw)